### DISCUSSION TITLE
Suspicious App!

### DISCUSSION THREAD

**Discussion by Chris-toph** [jely2002/youtube-dl-gui#481] [General]
Hello,
I have some suspicion about this app. It looks to change the proxy settings of the windows computer!
And it even manages to by-bass the restriction to change the setting which I've disable on windows.

**Comment by jely2002** [comment]
Hi Chris-toph,

I can assure you this app will not try to change your windows settings, as I've made it. The app does have its own settings for when you want to use a proxy, this has nothing to do with a setting you've disabled on windows. What do you mean by "to bypass the restriction to change the setting which I've disable on windows".

**Reply by Chris-toph** [reply]
Sorry, I should have put a question mark on the title.
In windows, one can restrict the permissions to change windows settings with the Microsoft Policy Editor app. And even the fact I've blocked the permission to change the proxy parameters, an app has changed it. 
I was using only another app at the same time which was Video Keeper from AceThinker (by the way, quite poor app).
The 3rd option would be an app which does it in background. But I do not have many softwares installed except the usual ones.
At least, thanks for your quick feedback.
Cheers,
